mbed_devices/mbed_devices.py

```diff
@@ -12,8 +10,13 @@
 
 def get_connected_devices() -> Iterable[Device]:
     """Returns Mbed Devices connected to host computer."""
-    with ThreadPoolExecutor() as executor:
```

Comment:
:D

Reply:
:trollface:

Reply:
Tbh, the complexity became immediately obvious when I had to handle exceptions in threads with a `map`. ;)